hyperon_das/das.py

```diff
@@ -686,13 +686,15 @@ def create_field_index(
 
     def fetch(
         self,
-        query: Union[List[dict], dict],
+        query: Optional[Union[List[dict], dict]] = None,
         host: Optional[str] = None,
         port: Optional[int] = None,
         **kwargs,
     ) -> Any:
         """
-        Fetch, from a DAS Server,  all links that match the passed query.
+        Fetch, from a DAS Server, all atoms that match the passed query.
```

Comment:
Docstrings aren't a free text. Theirs parts have special meanings. They can be used e.g. by external tools to build API docs. Sometimes such tools need to extract specific pieces to build sections or paragraphs in the target document. 

The first block of text in the docstring should be a brief, self-contained, summary of the function or class. You added part of this summary in a new paragraph, thus breaking the summary. That's why I'm suggesting to change it. My suggestion for this description is:

"Fetch, from a DAS Server, all atoms that match the passed query or all atoms in the server if None is passed as query."